pootle/apps/pootle_app/views/admin/dashboard.py

```diff
@@ -339,13 +113,20 @@ def rq_stats():
     return result
 
 
+def checks():
+    from django.core.checks.registry import registry
+
+    ret = []
```

Comment:
No, you need to flatten lists (sum lists), not sum the actual values of the lists themselves.